Dockerfile

```diff
@@ -1,12 +1,6 @@
 FROM node:14-alpine3.15
 MAINTAINER Professional Services <ps-dev@commercetools.com>
-
 WORKDIR /app
-
 COPY . /app
-
-RUN apk add --no-cache --virtual .gyp python3 make g++
```

Comment:
assuming those are needed for bunyan and we are not using it anymore.

Reply:
Interesting to see github action is working now without python installation. 
I added this after this failure, https://github.com/commercetools/commercetools-subscriptions/runs/5517795316?check_suite_focus=true

Reply:
It was working before like that: https://github.com/commercetools/commercetools-adyen-integration/blob/master/extension/Dockerfile

Reply:
Okay, I will check what was the problem in CD deployment script, thanks :)